c7n/resources/vpc.py

```diff
@@ -1220,33 +1220,53 @@ def process(self, resources):
             client.delete_security_group(GroupId=r['GroupId'])
 
 
-@SecurityGroup.action_registry.register('remove-permissions')
-class RemovePermissions(BaseAction):
-    """Action to remove ingress/egress rule(s) from a security group
+@SecurityGroup.action_registry.register('set-permissions')
+class SetPermissions(BaseAction):
+    """Action to update ingress/egress rule(s) of a security group
 
     :example:
 
     .. code-block:: yaml
 
             policies:
-              - name: security-group-revoke-8080
+              - name: modify-security-group
                 resource: security-group
                 filters:
-                  - type: ingress
-                    IpProtocol: tcp
-                    Ports: [8080]
+                  - or:
+                        - type: ingress
+                          IpProtocol: "-1"
+                          Ports: [22, 3389]
+                          Cidr: "0.0.0.0/0"
+                        - type: ingress
+                          IpProtocol: "-1"
+                          Ports: [22, 3389]
+                          CidrV6: "::/0"
                 actions:
-                  - type: remove-permissions
+                  - type: set-permissions
                     ingress: matched
+                    PermissionType: "ingress"
+                    IpProtocol: "TCP"
+                    FromPort: 22
+                    ToPort: 22
+                    Cidr: ["1.1.1.1/32","2.2.2.2/32","3.3.3.3./32"]
 
     """
     schema = type_schema(
-        'remove-permissions',
+        'set-permissions',
```

Comment:
we can't just remove the existing remove-permissions, that would break all existing/extant policies using remove-permissions.